todo.js

```diff
@@ -0,0 +1,17 @@
+console.dir(document)
+const addTaskBtn =  document.getElementById("addTask")
+const taskInputText =  document.getElementById("taskInput")
+const taskList =  document.getElementById("taskList")
+
+const handleSubmit = () => {
+    if(taskInputText.value){
+    const newTask = document.createElement("li");
+    newTask.innerHTML = taskInputText.value
```

Comment:
Let's modularize this a bit. 

Creating Elements should be another functions responsibility. We should only call the function here and the function would return the newly created element.

```js
const newTaskElement = createTaskElement(taskContent)
```